### PR TITLE
Lights, Chicken, Cow. Action!

### PR DESCRIPTION
**Pull request by KingOfThePing**
This PR removes some mismapped lights in the kitchen.

It also replaces the cow crate with a cow. Her name is Penelope. I also added a singular chicken. For chicken meat recipes. I hope one is enough without it being considered power creep(tm).

### CODE REVIEW

**Comment by KingOfThePing** [comment]
!bugfix

**Comment by KingOfThePing** [comment]
!review

**Comment by KingOfThePing** [comment]
I didnt know if I can still apply the bugfix tag, despite me adding one (1) chicken, too. Scold me if its wrong, please.

**Comment by TheDanseMacabre** [comment]
I'm not really sure why adding a named cow is a good idea considering that most rounds, the cow is killed and gibbed. It's livestock, not a pet.

**Comment by KingOfThePing** [comment]
> I'm not really sure why adding a named cow is a good idea considering that most rounds, the cow is killed and gibbed. It's livestock, not a pet.

For the same reason I forced Doxx to make all animals renamable. Because it's fun(ny).

**Comment by SleepyGemmy** [comment]
RE: [Comment.](https://github.com/Aurorastation/Aurora.3/pull/14406#issuecomment-1179391680)
\-----
> I didnt know if I can still apply the bugfix tag, despite me adding one (1) chicken, too. Scold me if its wrong, please.

this isn't a bugfix, since you're adding a new animal and renaming two.

RE: [Comment.](https://github.com/Aurorastation/Aurora.3/pull/14406#issuecomment-1179501386)
\-----
> I'm not really sure why adding a named cow is a good idea considering that most rounds, the cow is killed and gibbed. It's livestock, not a pet.

i second this. the chicken being named "Big Clucker" just feels very memey as well and doesn't make much sense.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
No more named animals. Get rid of the name. Also, if you want the bugfix tag, the chicken needs to go too.

**Comment by sheeplets** [comment]
The added chicken is nice, regardless of its name. Cargo isn't ever around to order from and the real cost in not just money but also your wasted time is prohibitive if all you want to do is serve some chicken sandwiches. Please keep that.

**Comment by KingOfThePing** [comment]
I am devastated, but as you wish.

File: maps/sccv_horizon/sccv_horizon-1_deck_1.dmm [hunk 1]
```diff
@@ -17629,6 +17629,7 @@
 /area/maintenance/operations)
 "olu" = (
 /obj/structure/flora/ausbushes/reedbush,
+/obj/item/holder/chicken,
```
Comment (alsoandanswer) [inline maps/sccv_horizon/sccv_horizon-1_deck_1.dmm:17632]: You added the chicken *item*, not a chicken mob.

**Review by alsoandanswer — CHANGES_REQUESTED** [review]

File: maps/sccv_horizon/sccv_horizon-1_deck_1.dmm (hunk 1 above)
Comment (KingOfThePing) [reply maps/sccv_horizon/sccv_horizon-1_deck_1.dmm:17632]: What in the goddamn

**Review by SleepyGemmy — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**